commands2/sysid/sysidroutine.py

```diff
@@ -0,0 +1,97 @@
+from dataclasses import dataclass
+from enum import Enum
+
+from wpilib.sysid import SysIdRoutineLog
+from ..command import Command
+from wpilib import Timer
+
+from wpimath.units import seconds, volts
+
+from typing import Callable, Optional
+
+
+volts_per_second = float
+
+
+class SysIdRoutine(SysIdRoutineLog):
+    @dataclass
+    class Config:
```

Comment:
Need docstrings. You can use https://github.com/auscompgeek/sphinxify to help convert them from Javadoc.